### PR TITLE
Fix issues with CI scripts

### PR DESCRIPTION
**Pull request by tfogo**
Not sure if this is the best place for `FEATURES=pure`. I figured putting it in the build target might be too prescriptive.

Please keep the v0.1 branch around.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,11 +22,11 @@ build.platform.all: build.platform.darwin \
 
 .PHONY: build.platform.darwin
 build.platform.darwin:
-	@$(MAKE) build PLATFORM=darwin_amd64
+	@$(MAKE) build PLATFORM=darwin_amd64 FEATURES=pure
```
Comment (kevindrosendahl) [inline Makefile]: i think you should add these here: https://github.com/mlab-lattice/lattice/blob/7a6e0926849e937485ce6634b11ca3b004faa254/ci/build-latticectl-binaries.sh#L24-L26

**Comment by tfogo** [comment]
@kevindrosendahl fixed

**Comment by tfogo** [comment]
One sec, let me switch those lines round

**Comment by tfogo** [comment]
Okay

**Pull request merged**